CarbKit/CarbStore.swift

```diff
@@ -110,7 +110,12 @@ public final class CarbStore: HealthKitSampleStore {
     public var carbRatioSchedule: CarbRatioSchedule?
 
     /// A trio of default carbohydrate absorption times. Defaults to 2, 3, and 4 hours.
-    public let defaultAbsorptionTimes: DefaultAbsorptionTimes
+    public var defaultAbsorptionTimes: DefaultAbsorptionTimes {
+        didSet {
+            self.maximumAbsorptionTimeInterval = defaultAbsorptionTimes.slow * 2
+            createQueries()
```

Comment:
Any existing queries should be stopped with healthStore.stop(query).  And self.queryAnchors should be cleared.